providers/implementations/signature/eddsa_sig.c

```diff
@@ -237,9 +232,9 @@ static int eddsa_digest_signverify_init(void *vpeddsactx, const char *mdname,
     return 1;
 }
 
-static int ed25519_digest_sign(void *vpeddsactx, unsigned char *sigret,
-                               size_t *siglen, size_t sigsize,
-                               const unsigned char *tbs, size_t tbslen)
+int ed25519_digest_sign(void *vpeddsactx, unsigned char *sigret,
+                        size_t *siglen, size_t sigsize,
+                        const unsigned char *tbs, size_t tbslen)
```

Comment:
Actually, `static` is perfectly fine, and isn't part of the indicator work per se, and should be preserved.

Reply:
This is a reversion of the previous commits -- no more or less than that.
Adding the static declarations back should be another PR after this is merged.

I.e. there weren't there before and shouldn't be there on reversion.